quinn-proto/src/connection.rs

```diff
@@ -928,6 +914,8 @@ where
         packet: Packet,
         remaining: Option<BytesMut>,
     ) -> Result<(), TransportError> {
+        let span = trace_span!("recv initial");
```

Comment:
Is it possible to chain the `.enter()` call, or does that run into lifetime issues?

Reply:
Sadly `creates a temporary which is freed while still in use` territory.